ci/do_ci.sh

```diff
@@ -25,13 +37,19 @@ if [ -n "$CIRCLECI" ]; then
         mv "${HOME:-/root}/.gitconfig" "${HOME:-/root}/.gitconfig_save"
         echo 1
     fi
-    export BAZEL_BUILD_OPTIONS="${BAZEL_BUILD_OPTIONS} --local_resources=4096,2,1"
-    export BAZEL_TEST_OPTIONS="${BAZEL_TEST_OPTIONS} --local_resources=4096,2,1 --local_test_jobs=4"
+    export BAZEL_BUILD_OPTIONS="${BAZEL_BUILD_OPTIONS} --jobs 8"
+    export BAZEL_TEST_OPTIONS="${BAZEL_TEST_OPTIONS} --jobs 8 --local_test_jobs=8"
+    export MAKEFLAGS="-j 8"
 fi
 
-export PATH=/usr/lib/llvm-7/bin:$PATH
-export CC=clang
-export CXX=clang++
+if [ "$1" != "coverage" ]; then
+    export PATH=/usr/lib/llvm-7/bin:$PATH
+    export CC=clang
+    export CXX=clang++
+else
```

Comment:
Nit: can you flip the logic here so we have `if coverage then .. else ..`, it's easier to parse.